Grace.py

```diff
@@ -0,0 +1,40 @@
+#A task was given before to just include a name and then parse it of which I did 
+#The second assignment was to parse both firstname and lastname together which was
+#done and Favour Samuel was gotten.
+#The third task isto Capitalise the first letters of "hello amazing grace"
+
+#Code for the second task.
+import argparse
+
+from argparse import ArgumentParser
+
+
+def hello(firstname, Lastname):
+    print( f'Hello {firstname} {Lastname}')
+
+
+if __name__ == '__main__':
+    args_to_parse = ArgumentParser()
+    args_to_parse.add_argument('--firstname', required=True, help='The name to greet')
+    args_to_parse.add_argument('--Lastname', required=True, help='The name to greet')

```

Comment:
I suggest to use `--Lastname`. And consider changing the help texts.

Reply:
I used --lastname. I do not know what to do concerning changing the help texts.

Reply:
Imagine a user who has no idea using your program. Then, does it help her/him to read that help? If I read the help text then I'd wonder why there are two arguments although it states that both are "the name to greet".